server/api/impl/proxy/auth.gogo

```diff
@@ -1,4 +1,4 @@
-package helloapp
+package http_hello
```

Comment:
This file will be gone once https://mattermost.atlassian.net/browse/MM-31117 is implemented, just keeping the code around